ldclient/event_consumer.py

```diff
@@ -13,11 +13,10 @@
 
 
 class EventConsumerImpl(Thread, EventConsumer):
-    def __init__(self, event_queue, sdk_key, config):
+    def __init__(self, event_queue, config):
```

Comment:
These changes necesitate a major release (due to changes in singleton init flow). Does this change your take on this?